File: workshop/csharp/src/CheckoutCS/Product.cs
using System;

namespace CheckoutCS
{
    public class Product
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public decimal Amount { get; set; }
        public long Version { get; set; }

        public Product(Guid id, string code, string description, decimal amount, long version) {
            Id = id;
            Code = code;
            Description = description;
            Amount = amount;
            Version = version;
        }
    }
}

File: workshop/php/src/Product.php
<?php

namespace CheckoutPHP;

class Product {
    /** @var string */
    private $id;
    /** @var string */
    private $code;
    /** @var string */
    private $description;
    /** @var float */
    private $amount;
    /** @var int */
    private $version;

    public function __construct(string $id, string $code, string $description, float $amount, int $version) {
        $this->id = $id;
        $this->code = $code;
        $this->description = $description;
        $this->amount = $amount;
        $this->version = $version;
    }

    public function getId() : int {
        return $this->id;
    }
}